frontend/admin/src/js/components/user/UserProfileDocument.tsx

```diff
@@ -112,7 +112,7 @@ export const UserProfileDocument = React.forwardRef<
         .accordion-header-context .icon {
           width: 1.5rem;
         }
-        .accordion-header-wrapper svg {
```

Comment:
Would this instance of the selector need to be changed to `.Accordion__Chevron` as well?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/1611e4794fbb981fb2639103a81da2a8bb8982a0/frontend/admin/src/js/components/poolCandidate/PoolCandidateDocument.tsx#L116-L118

Reply:
Thanks! I forgot we had this in two places. Updated with [update accordion icon class name](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4750/commits/22c16d304553f193772d3a723cca7600f8beb39f)